engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -42,12 +42,16 @@
                     <div class="g-block">
                         <div class="g-content">
                             <div class="g-array-item">
-                                {% if display.image.enabled and article.images.image_intro %}
-                                    <div class="g-array-item-image">
-                                        <a href="{{ article.route }}">
-                                            <img src="{{ url(article.images.image_intro) }}" {{ article.images.image_intro|imagesize|raw }} />
-                                        </a>
-                                    </div>
+                                {% if display.image.enabled %}
```

Comment:
Not sure what do you mean with
> to ensure that the image is available
To check that URL isn't blank or something else?